server/controllers/match.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+
+// add a match and return match ID
+function addMatch() {
+
+  var Match = mongoose.model('matches');
+  var match = new Match();
```

Comment:
In modern ES6, it's best to use const if a variable will not be redefined and let if it will be, you can replace all of these var

Reply:
removed var